app/views/layouts/_sidebar.html.slim

```diff
@@ -1,3 +1,12 @@
-div.sidebar.sidebar-left.sidebar-animate#site-navigation-sidebar class=sidebar_classes
-  ul.nav.navbar-stacked
-    = yield
+div class=sidebar_classes
+  div.container-fluid
+    div.navbar-header
+      button.btn.btn-default.pull-right.navbar-toggle type="button" data-toggle="collapse" data-target="#site-navigation-sidebar" aria-expanded="false" aria-controls="navbar"
+        span.sr-only
+          = t('layout.navbar.toggle_sidebar')
+        span.glyphicon.glyphicon-chevron-down aria-hidden='true'
+      = render partial: 'layouts/user', locals: { user: current_user }
```

Comment:
That area is the navbar header. User layout seems to fit nicely for now cause it has a profile pic and a link to user. 

Will probably replace this when I implement the user badge. Course users should see a badge, and admins see something more basic.